eng/Signing.props

```diff
@@ -1,5 +1,14 @@
 <Project>
 
+  <PropertyGroup>
+    <Product>Sdk</Product>
+    <BlobStoragePartialRelativePath>$(Product)</BlobStoragePartialRelativePath>
+    <BlobStoragePartialRelativePath Condition="'$(IsNotOrchestratedPublish)' == 'false'">assets/$(Product)</BlobStoragePartialRelativePath>
```

Comment:
`IsNotOrchestratedPublish` is a dead property, isn't it?

Reply:
Yep. Looks like it's been dead since the dotnet/cli->dotnet/sdk consolidation.